azext_edge/edge/_help.py

```diff
@@ -703,6 +703,98 @@ def load_iotops_help():
             az iot ops dataflow list -p myprofile --in mycluster-ops-instance -g myresourcegroup
     """
 
+    helps[
+        "iot ops registry"
+    ] = """
+        type: group
+        short-summary: Manage container registry endpoints.
+    """
+
+    helps[
+        "iot ops registry add"
+    ] = """
+        type: command
+        short-summary: Add a container registry endpoint to an instance.
+        long-summary: |
+          Only Azure Container Registry (ACR) endpoints are supported.
+
+        examples:
+        - name: Add a basic anonymous registry endpoint
```

Comment:
tiny nit, being consistent about ending sentences with periods even in examples.